### PR TITLE
Travis: Run apt update before calling apt install

### PR DESCRIPTION
**Pull request by g2p**

### CODE REVIEW

File: .travis-ci.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 eval `opam config env`
+sudo apt update
```
Comment (samoht) [inline .travis-ci.sh]: normally the `depext -u` below should run `apt update`:

```
$ opam depext --help

    -u, --update
           Update the OS package sets before installation

```
Do you have some logs where this is broken?

Comment (g2p) [inline .travis-ci.sh]: Oh, good point. Here's my log: https://travis-ci.org/mirage/mirage/jobs/244219628

Comment (g2p) [inline .travis-ci.sh]: Looks like it should be added to mirage-skeleton depext calls then.

Comment (g2p) [inline .travis-ci.sh]: Oh wait, it has to be added to a `make depend` target that's generated by mirage itself.

Comment (g2p) [inline .travis-ci.sh]: See #841. It's a more intrusive change but probably the right thing to do.

**Comment by g2p** [comment]
Running apt update is currently broken due to the Stretch release (testing is now an alias for buster and not stretch, and that breaks things).
Probably something that needs to be fixed at the Docker image level.

**Comment by avsm** [comment]
I'm taking a look at the Docker base images.

**Comment by samoht** [comment]
Closing as the other PR is now in.

**Pull request closed without merging**